content/spin/cli-reference.md

```diff
@@ -3241,19 +3121,25 @@ TRIGGER OPTIONS:
             used. The cert should be in PEM format
             
             [env: SPIN_TLS_CERT=]
+            Only appplies to HTTP triggers
 
         --tls-key <TLS_KEY>
             The path to the certificate key to use for https, if this is not set, normal http will
             be used. The key should be in PKCS#8 format
             
             [env: SPIN_TLS_KEY=]
+            Only appplies to HTTP triggers
+
 ```
 
 {{ blockEnd }}
 
 {{ blockEnd }}
 
-### Watch
+> **Please note:** If the `-f` or `--from` options do not accurately infer the intended registry or `.toml` file for your application, then you can explicitly specify either the `--from-registry` or  `--from-file` options to clarify this.
```

Comment:
It's explained in an error message if the inference doesn't work.  I'm not hugely enthusiastic about making the help text even longer.

The review comment, though, was in reference to if we were going to manually edit help text body anyway (for the HTTP stuff).  Since that's no longer the plan, I think this comment goes away and we should leave this as is.